### ISSUE TITLE
[3.3] Bug in templates load order.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#6479]
It seems the preferred order of where templates are found has a bug. `app/theme_defaults` gets preference over the current theme folder, while it shouldn't. 

Details
-------

 - Relevant Bolt Version: [ 3.3 | master ]

Reproduction
------------

Create two `test.twig` files: 

```
echo "[foo]" >> theme/base-2016/test.twig
echo "[bar]" >> app/theme_defaults/test.twig
```

Now, in index.twig, add: 

```
Test is {{ include("test.twig") }}
```

The expected result is "Test is [foo]". The actual result is "Test is [bar]". 

@CarsonF I think this was introduced recently in the work you did on Twig rendering. Could you look into this?

**Comment by CarsonF** [comment]
Yes that's a derp. [These](https://github.com/bolt/bolt/blob/release/3.3/src/Provider/TwigServiceProvider.php#L302-L303) should be `prependPath` not `addPath`.

**Comment by bobdenotter** [comment]
Ah, right.. You on it? or should I PR the fix? :-)

**Comment by CarsonF** [comment]
Go ahead :)

**Comment by bobdenotter** [comment]
Ok, Thanks for the pointer! 👍

**Comment by GawainLynch** [comment]
Fixed in #6480
